hazelcast-client/src/test/java/com/hazelcast/client/cache/nearcache/ClientCacheNearCachePreloaderSlowTest.java

```diff
@@ -17,32 +17,31 @@
 package com.hazelcast.client.cache.nearcache;
 
 import com.hazelcast.config.InMemoryFormat;
-import com.hazelcast.test.HazelcastParametersRunnerFactory;
+import com.hazelcast.test.HazelcastParallelParametersRunnerFactory;
+import com.hazelcast.test.annotation.ParallelTest;
 import com.hazelcast.test.annotation.SlowTest;
 import org.junit.Before;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+import org.junit.runners.Parameterized.UseParametersRunnerFactory;
 
 import java.util.Collection;
 
 import static java.util.Arrays.asList;
 
+/**
+ * This test cannot be a {@link ParallelTest}, since it uses the same storage file as {@link ClientCacheNearCachePreloaderTest}.
```

Comment:
There are test methods which read from fixed files, which can only be locked once per JVM:
```
com.hazelcast.core.HazelcastException: Cannot acquire lock on /home/david/IdeaProjects/hazelcast/hazelcast-client/target/test-classes/nearcache-negative-fileformat/nearCache-defaultNearCache.store.lock. File is already being used by another Hazelcast instance.
```
```java
    private final File preloadDir10kInt = getFileFromResources("nearcache-10k-int");
    private final File preloadDir10kString = getFileFromResources("nearcache-10k-string");
    private final File preloadDirEmpty = getFileFromResources("nearcache-empty");
    private final File preloadDirInvalidMagicBytes = getFileFromResources("nearcache-invalid-magicbytes");
    private final File preloadDirInvalidFileFormat = getFileFromResources("nearcache-invalid-fileformat");
    private final File preloadDirNegativeFileFormat = getFileFromResources("nearcache-negative-fileformat");
```
So we cannot execute two instances of an `AbstractNearCachePreloaderTest` at the same time. And since e.g. `ClientMapNearCachePreloaderTest` and `ClientMapNearCachePreloaderSlowTest` are in the same package, the chance is high, that they are executed at the same time in the Sonar build (which runs `QuickTest` and `SlowTest` together).